### PR TITLE
Add Documentation about severity and priority of Issues/PRs

### PR DESCRIPTION
**Pull request by okorshenko**
## Purpose of this pull request

magento/devdocs#7074: Add Documentation about severity and priority of Issues

On magento/magento2 project there are lots of issue reports and pull requests in the queue. It is clear, that all these submissions can't be processed immediately. At the same time, these contributions bring different value to the product and community. Thus, it is important to stay focused on the most critical issues and pull requests and make sure that they are addressed in time. Less critical issues and pull requests can be processed later.

Having clear contribution rules in place will help build clear expectations for community contributors and establish clear priorities for Community maintainers and Magento team.

## Affected DevDocs pages

- src/guides/v2.3/contributor-guide/contributing.md

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by dobooth** [comment]
running tests

File: src/guides/v2.3/contributor-guide/contributing.md [hunk 1]
```diff
@@ -27,13 +28,70 @@ We use the [fork and pull](#forks-and-pull-requests) model to contribute to the
...
 
-The Community Engineering Team reviews all issues and contributions submitted by the community developers. During the review we might require clarifications from the contributor. If there is no response from the contributor in two weeks (14 days) time, the issue is closed.
+The Community Engineering Team reviews all issues and contributions submitted by the community developers. During the review we might require clarifications from the contributor. If there is no response from the contributor in two weeks (14 days) time, the issue might be closed.
 
 When the Community Engineering Team works on reviewing the suggested changes, we will add a label to the issue to indicate certain information, like the status or who is working the issue. See [Labels](#labels) to learn more.
 
 {:.bs-callout-tip}
 Refer to [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca) for detailed information about the License Agreement. All contributors are required to submit a click-through form to agree to the terms.
 
+## Community Backlog Priority{#backlog}
```
Comment (dobooth) [inline src/guides/v2.3/contributor-guide/contributing.md:38]: Please put a space between the heading and the anchor name.

Comment (okorshenko) [inline src/guides/v2.3/contributor-guide/contributing.md:38]:
```suggestion
## Community Backlog Priority {#backlog}
```

Comment (okorshenko) [reply src/guides/v2.3/contributor-guide/contributing.md]: fixed

**Comment by dobooth** [comment]
running tests

**Pull request merged**